### PR TITLE
Scala 2.13

### PR DESCRIPTION
**Pull request by cquiroz**
This PR adds support to cross compile and moves publication to use sbt-ci-release

### CODE REVIEW

File: .github/workflows/scala.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: olafurpg/setup-scala@v2
+        with:
+          java-version: 1.8
+      - name: Run compile
+        run: csbt +compile
```
Comment (rpiaggio) [inline .github/workflows/scala.yml]: Why not use `csbt +test` here?

Comment (cquiroz) [reply .github/workflows/scala.yml]: There aren't tests on this package

**Comment by cquiroz** [comment]
@rpiaggio Good catch, the README is now obsolete

File: .github/workflows/scala.yml (hunk 1 above)
Comment (rpiaggio) [reply .github/workflows/scala.yml]: I was thinking that in this case `test` will do the same as `compile`, but we won't have to change it if we ever implement tests.

**Pull request merged**